salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
Here's the output of my grains(the section that I'm after):
```yaml
    locale_info:
        ----------
        defaultencoding:
            UTF-8
        defaultlanguage:
            pt_PT
        detectedencoding:
            UTF-8
```

Yours doesn't output `detectedencoding`? or you just didn't include it?